wgpu-hal/src/gles/shader_clear.vert

```diff
@@ -0,0 +1,11 @@
+#version 300 es
+precision lowp float;
+// A triangle that fills the whole screen
+const vec2[3] TRIANGLE_POS = vec2[](
+  vec2( 0., -3.),
```

Comment:
I have a small personal vendetta against omitting the trailing `0` from floats. Could you add them back :)